File: src/test/java/model/TicketTest.java
package model;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import java.time.LocalDate;

import static org.junit.jupiter.api.Assertions.*;

class TicketTest {
    private Ticket ticket;
    private Ticket ticket0;

    @BeforeEach
    void setUp() {
        ticket = new Ticket(new Performance(1L, LocalDate.now(), "Cluj-Napoca", 206, 1678, "Kygo"), "owner", 3);
        ticket0 = new Ticket(1L, new Performance(1L, LocalDate.now(), "Cluj-Napoca", 206, 1678, "Kygo"), "owner", 33);
    }

    @Test
    void getPerformance() {
        assertEquals(ticket.getPerformance().getID(), 1L);
    }

    @Test
    void setPerformance() {
        ticket.setPerformance(new Performance(2L, LocalDate.now(), "Cluj-Napoca", 206, 1678, "Kygo"));
        assertEquals(ticket.getPerformance().getID(), 2L);
    }

    @Test
    void getOwnerName() {
        assertEquals(ticket.getOwnerName(), "owner");
    }

    @Test
    void setOwnerName() {
        ticket.setOwnerName("name");
        assertEquals(ticket.getOwnerName(), "name");
    }

    @Test
    void getNoOfSeats() {
        assertEquals(ticket.getNoOfSeats(), 3);
    }

    @Test
    void setNoOfSeats() {
        ticket.setNoOfSeats(2);
        assertEquals(ticket.getNoOfSeats(), 2);
    }
}

File: Tests/model/TicketTest.cs
using System;
using MusicFestivalMPP.model;
using NUnit.Framework;

namespace Tests.model;

public class TicketTest
{
    private Ticket _ticket; 
    
    [SetUp]
    public void Setup()
    {
        _ticket = new Ticket(
            1, new Performance(1, new DateTime(2022, 8, 26), "CJ", 100, 200, "artist"), "owner", 3
        );
    }

    [Test]
    public void GetPerformance()
    {
        Assert.AreEqual(_ticket.Performance.Id, 1);
    }

    [Test]
    public void SetPerformance()
    {
        _ticket.Performance.Id = 2;
        Assert.AreEqual(_ticket.Performance.Id, 2);
    }
    
    [Test]
    public void GetOwnerName()
    {
        Assert.AreEqual(_ticket.OwnerName, "owner");
    }

    [Test]
    public void SetOwnerName()
    {
        _ticket.OwnerName = "name";
        Assert.AreEqual(_ticket.OwnerName, "name");
    }
    
    [Test]
    public void GetNoOfSeats()
    {
        Assert.AreEqual(_ticket.NoOfSeats, 3);
    }

    [Test]
    public void SetNoOfSeats()
    {
        _ticket.NoOfSeats = 2;
        Assert.AreEqual(_ticket.NoOfSeats, 2);
    }
}